docs/develop/pivottables.md

```diff
@@ -0,0 +1,152 @@
+
+```typescript
+  farmPivot.addRowHierarchy(farmPivot.getHierarchy("Farm"));
+  farmPivot.addRowHierarchy(farmPivot.getHierarchy("Type"));
+  farmPivot.addDataHierarchy(farmPivot.getHierarchy("Crates Sold at Farm"));
+  farmPivot.addDataHierarchy(farmPivot.getHierarchy("Crates Sold Wholesale"));
+```
+
+:::image type="content" source="../images/pivottable-data-hierarchy.png" alt-text="A PivotTable showing the total sales of different fruit based on the farm they came from.":::
+
+## Layout ranges
+
+Each part of the PivotTable maps to a range. This lets your script get data from the PivotTable for use later in the script or to be returned in a [Power Automate flow](power-automate-integration.md). These ranges are accessed through the [PivotLayout](/javascript/api/office-scripts/excelscript/excelscript.pivotlayout) object acquired from `PivotTable.getLayout()`. The following diagram shows the ranges that are returned by the methods in `PivotLayout`.
+
+:::image type="content" source="../images/pivottable-layout-breakdown.png" alt-text="A diagram showing which sections of a PivotTable are returned by the layout's get range functions.":::
+
+## Filters and slicers
+
+There are three ways to filter a PivotTable.
+
+- [FilterPivotHierarchies](/javascript/api/office-scripts/excelscript/excelscript.filterpivothierarchy)
+- [PivotFilters](/javascript/api/office-scripts/excelscript/excelscript.pivotfilters)
+- [Slicers](/javascript/api/office-scripts/excelscript/excelscript.slicer)
+
+### FilterPivotHierarchies
+
+`FilterPivotHierarchies` add an additional hierarchy to filter every data row. Any row with an item that is filtered out is excluded from the PivotTable and its summaries. Since these filters are based on items, they only work on discrete values. If "Classification" is a filter hierarchy in our sample, users can select the values of "Organic" and "Conventional" for the filter. Similarly, if "Crates Sold Wholesale" is selected, the filter options would be the individual numbers, such as 120 and 150, instead of numerical ranges.
+
+FilterPivotHierarchies are created with all values selected. This means that nothing is filtered until the user manually interacts with the filter control or a `PivotManualFilter` is set on the field belonging to the `FilterPivotHierarchy`.
+
+This following code snippet adds "Classification" as a filter hierarchy.
+
+```typescript
+  farmPivot.addFilterHierarchy(farmPivot.getHierarchy("Classification"));
+```
+
+:::image type="content" source="../images/pivottable-filter-hierarchy.png" alt-text="A filter control that uses 'Classification' for a PivotTable.":::
+
+### PivotFilters
+
+The `PivotFilters` object is a collection of filters applied to a single field. Since each hierarchy has exactly one field, you should always use the first field in `PivotHierarchy.getFields()` when applying filters. There are four filter types.
+
+**Date Filter**: Calendar date-based filtering.
```

Comment:
These filters are displaying as a paragraph, but I suspect they are intended to display as a list. Also, should both words be capitalized? Suggesting these changes for all 4 filter types. 
```suggestion
- **Date filter**: Calendar date-based filtering.
```